src/main/java/no/cantara/realestate/mappingtable/MappingKey.java

```diff
@@ -2,4 +2,7 @@
 
 public interface MappingKey<T> {
     T getKey();
+    boolean equals(Object o);
```

Comment:
These should be removed. hashCode and equals can always be overridden anyway and do never need to be specified in an interface.